string robot_id
float32 x
float32 y
string status
bool paused